CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ Updated native SDK versions:
 
 ### App Center Push
 
+* **[Bug fix]** Fix receiving push notifications from background.
```

Comment:
This needs to be move to a new version (might need to catch up with release changes before).